lib/PseudoFactory.js

```diff
@@ -162,8 +164,8 @@ var PseudoFactory = function(options) {
             });
             break;
 
-        case 'debug-asian':
-            logger.trace('Debug rtl pseudo for locale ' + options.targetLocale);
+        case 'debug-han-simplified':
+            logger.trace('Debug simplified-chinese pseudo for locale ' + options.targetLocale);
```

Comment:
Same here

Reply:
I fixed it. and add a test case and updated release note as well.